config/file-size.js

```diff
@@ -0,0 +1,79 @@
+const { basename, normalize } = require("path");
+const { readFile: readFileCb } = require("fs");
+const { promisify } = require("util");
+const readFile = promisify(readFileCb);
+
+const kolor = require("kleur");
+const prettyBytes = require("pretty-bytes");
+const brotliSize = require("brotli-size");
+const gzipSize = require("gzip-size");
```

Comment:
let's show file size without gzip

it is more frightening this way and changes reflects much more visible

And can we write the bundle size to some file in the git repo on some hook? So we will know how will a new change effect the bundle size.

Reply:
@goooseman Done.